earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
@andreisel If you take a look at the fusion_installer branch, you'll see that the src/installer directory is actively in use. I guess it doesn't matter whether the directories are under src or not, but I like the idea of separating legacy scripts into legacy_installer and in-use scripts into installer. Is your suggestion to move both directories into the earth_enterprise directory? @arjntc1 do you have an opinion? We'll need to coordinate all this because fusion_installer is in active development.

@bradh I believe Aaron-installer refers to the changes Aaron has made on fusion_installer.